### PR TITLE
Dicom metadata

### PR DESCRIPTION
**Pull request by htried**
Added simple notebook to look at some metadata

### CODE REVIEW

File: etl/lib/mip.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    handler.setFormatter(formatter)
+    root_logger.addHandler(handler)
+
+
+def mip_array(array: np.ndarray, type: str) -> np.ndarray:
+    print(np.max(array, axis=0).shape)
+    return np.max(array, axis=0)
+
+
+def crop(arr: np.ndarray, whence: str):
```
Comment (therladbsgh) [inline etl/lib/mip.py]: Im wondering if this file should be split. One for transformations (which would be better fit for transforms.py), and one for GCS management (download / uploading files). Maybe one more for __main__.

File: etl/lib/mip.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+if __name__ == '__main__':
+    configure_logger()
+    client = authenticate()
+    bucket = client.get_bucket('elvos')
+
+    # from numpy directory
+    # for in_blob in bucket.list_blobs(prefix='numpy/'):
+    #
+    #     # blacklist
+    #     if in_blob.name == 'numpy/LAUIHISOEZIM5ILF.npy':
```
Comment (therladbsgh) [inline etl/lib/mip.py]: Might want to remove commented-out code if we won't use them

File: etl/lib/multichannel_mip.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
+"""
```
Comment (therladbsgh) [inline etl/lib/multichannel_mip.py]: May be better merged with mip.py. Works better if mip.py is split into transforms / GCS handling / etc. And then multichannel_mip.py uses those same files.

File: labels.csv [hunk 4]
```diff
@@ -0,0 +1,906 @@
+patient_id,label
```
Comment (therladbsgh) [inline labels.csv]: Not sure if this is something to commit

File: ml/models/simple_unet.py [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+        deconv4_1 = Cropping2D(((0, 0), (1, 1)))(conv1)
+        both_4 = Concatenate([deconv4, deconv4_1])
+
+        # Fully connected layers
+        dense1 = Dense(1024, activation='relu', use_bias=True)(both_4)
+        dense2 = Dense(1024, activation='relu', use_bias=True)(dense1)
+        output_img = Dense(num_classes, activation='sigmoid',
+                                    use_bias=True)(dense2)
+
+        # begin unused code
```
Comment (therladbsgh) [inline ml/models/simple_unet.py]: Remove commented code

File: ml/models/simple_unet.py [hunk 6]
```diff
@@ -0,0 +1,152 @@
+"""
+A simple 3D Convolutional network. It has 5 convolutional
```
Comment (therladbsgh) [inline ml/models/simple_unet.py]: Reminds me that we have to eventually change all our model documentation.. lol

**Comment by luke-zhu** [comment]
Can you fix the flake8 errors: https://travis-ci.com/elvoai/elvo-analysis/builds/76605352?utm_source=github_status&utm_medium=notification

**Comment by luke-zhu** [comment]
The notebook is pretty large, so it would be good to clear the outputs. In general, the code looks pretty clean but we want to improve our code quality more over time.

**Comment by therladbsgh** [comment]
LGTM

**Pull request merged**